examples/id-filter-example.py

```diff
@@ -0,0 +1,52 @@
+from aiogram import Bot, Dispatcher, executor, types
```

Comment:
By the way the filename with hyphens is bad for this case. Please rename this file with replacing hyphens to underscores.

`id-filter-example.py` -> `id_filter_example.py`

Reply:
Done with [80b1168](https://github.com/aiogram/aiogram/pull/172/commits/80b11684801446a4e457214738adde4be1018ccd)